packages/vue/src/huicharts/huicharts-core/src/chart-core.ts

```
@@ -243,14 +243,14 @@
     },
 
     size: {
-      handler(val) {
+      handler() {
         this.$nextTick(() => {
           this.integrateChart && this.integrateChart.echartsIns && this.integrateChart.echartsIns.resize()
         })
       }
     },
     colors: {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)